include/mbedtls/build_info.h

```diff
@@ -221,6 +221,47 @@
 #define MBEDTLS_PK_PARSE_C
 #endif
 
+/* Helpers to state that each key is supported either on the builtin or PSA side. */
+#if defined(MBEDTLS_ECP_DP_SECP521R1_ENABLED) || defined(PSA_WANT_ECC_SECP_R1_521)
+#define MBEDTLS_HAVE_SECP521R1
```

Comment:
This name feels like it's missing a namespace. `MBEDTLS_ECP_HAVE_SECP521R1`, I guess, since the scope is ECP-light?